index.rst

```diff
@@ -6,21 +6,44 @@
 
+In order to achieve widespread use of driving simulators for function developers, the connection between the function development framework and the simulation environment has to rely on generic and standardized interfaces.
+An easy and straight-forward compatibility between automated driving functions and the variety of available driving simulation frameworks is made possible by OSI.
+OSI was developed exactly for this purpose and addresses the emerging standard ISO 23150 for real sensors' standardized communication interface.
 In this context, OSI defines generic interfaces to ensure modularity, integrability, and interchangeability of the individual components:
+The following figure shows the interfaces and models involved in modeling a sensor.
 
-.. image:: _static/images/osicontextwiki.png
+.. image:: _static/images/osi-context.png
    :align: center
+   :alt: Open Simulation Interface overview
 
+OSI also defines interfaces for traffic participant models.
+These interfaces allow to send commands to traffic participant models and to receive their updated state.
+The following figure shows the interfaces of a generic traffic participant.
 
-The Vision
------------
-In order to achieve widespread use of driving simulators for function developers, the connection between the function development framework and the simulation environment has to rely on generic and standardized interfaces. An easy and straightforward compatibility between automated driving functions and the variety of available driving simulation frameworks is made possible by OSI. OSI was developed exactly for this purpose and addresses the emerging standard ISO 23150 for real sensors' standardized communication interface.
+.. image:: _static/images/osi-traffic-participant-principle.png
+   :align: center
+   :alt: Traffic participant interfaces
+
+Traffic participant models may use other OSI interfaces internally, for example, to model autonomous vehicles.
+The following figure shows a more advanced use case for traffic participants.
+
+.. image:: _static/images/osi-traffic-participant-advanced.png
+   :align: center
+   :alt: Traffic participant with sensor models, AD function, and dynamic model
+
+OSI currently provides only limited support for data structures describing measured internal states of the traffic participant.
+Actuator intentions are currently not covered by OSI and must be handled with a different data description format.
+
+Interfaces
+----------
 
 OSI contains an object-based environment description using the message format of the `protocol buffer <https://github.com/protocolbuffers/protobuf/wiki>`_ library developed and maintained by Google. OSI defines top-level messages that are used to exchange between separate models. Top-level messages defines the ``GroundTruth`` interface, the SensorData interface and, since OSI version 3.0.0, the ``SensorView`` / ``SensorView`` configuration interfaces and the ``FeatureData`` interface.
```

Comment:
I think SensorData should be in in quotes as well. Also SensorViewConfiguration should be in quotes (further mentions below this comment).

Reply:
- [ ] Put SensorData in backticks